src/routes/styleguide/__layout.reset.svelte

```diff
@@ -0,0 +1,25 @@
+<script context="module" lang="ts">
+  import { mode } from '$app/env';
+
+  /** @type {import('@sveltejs/kit').Load} */
+  export async function load({}) {
+    let responseObj: any = {
+      status: 200,
+    };
+
+    if (mode !== 'development') {
+      responseObj = {
+        status: 404,
+        error: new Error('Could not find page'),
+      };
+    }
+
+    return responseObj;
+  }
+</script>
+
+<script lang="ts">
+  import '../../app.postcss';
```

Comment:
Do we have a path alias for this?

Reply:
I was copying what we were doing in our main __layout page but it seems that we get this injected another way. I deleted this import and all the tailwind css worked fine.